docs/core/install/localized-intellisense.md

```diff
@@ -28,19 +26,20 @@ ms.date: 12/18/2019
 
 1. Extract the contents of the zip file.
 
-1. Navigate to the .NET Core installation folder. By default, it's under *%ProgramFiles%\dotnet\packs*.
+1. Navigate to the moniker folder.
```

Comment:
The steps below don't explain how to navigate to the .NET Core installation folder. They explain how to navigate to the moniker folder. The first step in navigating to the moniker folder is to navigate to the .NET Core installation folder.

Reply:
Not sure if I follow.

Reply:
This is what was before:

![image](https://user-images.githubusercontent.com/31348972/72317340-a0956000-36a1-11ea-92a5-ad820ca92212.png)

For me, I'll interpret it as if these red colored steps explain how to navigate to %ProgramFiles%\dotnet\packs, but they navigate to something like %ProgramFiles%\dotnet\packs\Microsoft.NETCore.App.Ref\3.1.0\ref\netcoreapp3.1

Reply:
I feel like this would be better if there was a path template up front: `%ProgramFiles%\dotnet\packs\[sdk-path]\[sdk-version]\ref\[sdk-moniker]`. Then define each parameter and give an example path. 

Also, we need to get this update for VS for Mac.

Reply:
That's exactly where I wanted you to land:
%ProgramFiles%\dotnet\packs\Microsoft.NETCore.App.Ref\3.1.0\ref\netcoreapp3.1

So the original steps are correct.

Reply:
I see where the confusion is. I think the only change I'd make is perhaps change line 31 to say something like

1. Navigate to the .NET Core Intellisense folder.

And then add a new line for opening the program files location before "Choose which SDK you want to install the IntelliSense for..."

- Navigate to the .NET Core installation folder. By default, it's under *%ProgramFiles%\dotnet\packs*.